src/index.html

```diff
@@ -1,13 +1,78 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
```

Comment:
Styling using classes is good, though `html` element is usually an exception, its ok to style it by tag
```css
html {
  font-family: ...
}
```

Reply:
Same for `body`

Reply:
Thx 😊 moved to global.scss. Done ✅